scania-digital-design-system/tegel
# feat(table): zebra rows & columns

## **Describe pull-request**  
Adding "zebra" feature to table rows & columns

## **Issue Linking:**  
_Choose one of the following options_
- **Jira:** Add ticket number after `CDEP-`: [CDEP-3471](https://tegel.atlassian.net/browse/CDEP-3471)    
- **Jira:** Add ticket number after `CDEP-`: [CDEP-3472](https://tegel.atlassian.net/browse/CDEP-3472) 

## **How to test**  
1. Open preview link
2. Check Tables - > Zebra Mode
3. Check two different modes for rows and columns
4. Compare it with[ Figma design](https://www.figma.com/design/f0cqxOO7o859GNRkBKfEIx/Data-tables-exploration?node-id=1465-66217&m=dev)

## **Checklist before submission**

## **Suggested test steps**


## **Additional context**  

[CDEP-3471]: https://tegel.atlassian.net/browse/CDEP-3471?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[CDEP-3472]: https://tegel.atlassian.net/browse/CDEP-3472?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
